sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -0,0 +1,280 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
+
+from __future__ import annotations
```

Comment:
For some reason lint is failing. 

Seeing 15:48:37 apache_beam/ml/inference/tensorrt_inference.py:20: error: Module '__future__' has no attribute 'annotations'  [attr-defined]

Trying https://github.com/apache/beam/pull/22804/files to see if it fixes it.

Reply:
Tbh, not sure why it has to be there. I think it was either added for some internal beam purposes or maybe I just left it there from pytorch implementation. Thanks for looking into it @tvalentyn

Reply:
I added this when I [refactored](https://github.com/apache/beam/pull/22131/commits/a79e91ffec286881a5e64b272496b1ad8350b746) the `cuda` and `tensorrt` imports inside the functions. Yeah, it looks to be the `3.6` version that doesn't support `__future__`. `3.7` should work. (It was also misleading when trying to debug because in `tox.ini`, our mypy test is called `py37-mypy`, which initially made me think we already have `3.7`)